dwc_server.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import tensorflow as tf
+import os
+#import asyncio
+from fastapi import FastAPI, File, UploadFile
+from fastapi.responses import FileResponse
+#import websockets
+import copy
+import numpy as np
+import uuid
+
+app = FastAPI()
+
+datasets = []
+num_clients = 1
+model = None
+priors = []
+client_ids = []
```

Comment:
remove any globals that are not necessary and pass things around in the functions.